lfs/client.go

```diff
@@ -595,3 +603,25 @@ func sendApiEvent(event int) {
 	default:
 	}
 }
+
+type notImplError struct {
+	error
+}
+
+func (e notImplError) NotImplemented() bool {
+	return true
+}
+
+func newNotImplError() error {
+	return notImplError{errors.New("Not Implemented")}
+}
+
+func isNotImplError(err *WrappedError) bool {
+	type notimplerror interface {
+		NotImplemented() bool
+	}
+	if e, ok := err.Err.(notimplerror); ok {
+		return e.NotImplemented()
+	}
```

Comment:
Since the error itself has a static string, what about something like this:

```go
var notImplementedErr = errors.New("Not Implemented")

func newNotImplError() *WrappedError {
  return Error(notImplementedErr)
}

func isNotImplError(err *WrappedError) bool {
  if err == nil {
    return false
  }

  return err.Err == notImplementedErr
}